patch/README.md

```diff
@@ -1,21 +1,40 @@
 # patch command line tool
 
-This is the tool used by the `shorebird` command line to compute the patch
-file for uploading to the server.
-
-This currently uses the rust `bidiff` crate to compute the patch file.
-and could just use the `bic` command line tool included in that crate. However
-we're explicitly writing our own command line to allow us to change the
-underlying compression without affecting the `shorebird` command line callers.
+This is the tool used by the `shorebird` command line to package up a patch
```

Comment:
We might even just say this creates a binary diff (I believe that's all it does?)

Reply:
nvm, answered later